tests/crypto/test_ccm_mode/testcase.yaml

```diff
@@ -0,0 +1,3 @@
+tests:
+-   test:
+        tags: crypto aes ccm
```

Comment:
That will lead to magic behavior nobody understands. "Explicit is better than implicit".